src/main/scala/com/microsoft/azure/cosmosdb/spark/config/CosmosDBConfig.scala

```diff
@@ -153,19 +154,19 @@ object CosmosDBConfig {
   val DefaultQueryMaxRetryWaitTimeSecs = 1000
   val DefaultSamplingRatio = 1.0
   val DefaultPageSize = 1000
-  val DefaultSampleSize = DefaultPageSize
+  val DefaultSampleSize: Int = DefaultPageSize
```

Comment:
I don't think you need to define the type. Any reason we are doing this?

Reply:
IMO readability is the big one.  If I don't have the project open in an IDE, I wouldn't know if that's an integer, string, or float.  The library's config is all Map[String,String] but those values are sometimes mapped to Ints and it's easy to do the wrong thing accidentally.  

When I set config today, I have to open up the defaults to understand what type a particular setting should be in.

Reply:
A bad example of this, perhaps, is this block of code chained together.  If I wasn't using an IDE, I would have a really hard time keeping track of the types and I would be going line to by with types declared to assert that the transformations are doing what I expect them to.

![Untitled](https://user-images.githubusercontent.com/15852038/80241758-05eddb00-8619-11ea-9ce5-a59351979bf3.png)

Reply:
Sometimes you also _dont_ want the compiler inferring types.

```scala
val a = List("abc")
val b = List("abc, 1)

val c: List[String]= List("abc", 2)
```

`a` is a `List[String]` but `b` would be a `List[Any]` which might not be what I want/expect.  `c` would give me a compile time error.

Reply:
+1